CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## v0.1.1 - 2018-08-30
```

Comment:
🏆 

I wonder tho if this could be considered a breaking change and technically deserve a major version bump...

Reply:
Yeah ok I'll cave